src/rating/Rating.js

```diff
@@ -69,28 +69,40 @@ export default class Rating extends Component {
         this.setState({ value: gesture.dx });
       },
       onPanResponderRelease: event => {
-        onFinishRating(this.getCurrentRating());
+        const rating = this.getCurrentRating();
+        if (!fractions) {
+          // "round up" to the nearest star/rocket/whatever
+          this.setCurrentRating(rating);
```

Comment:
For instance, `<Rating showRating />`:
* Without the `if` block, the 4th star is only partially filled even though the rating is at 4:
![before](https://cloud.githubusercontent.com/assets/241553/26780751/32f7be4c-49fc-11e7-8313-58c76266b7c8.gif)

I'm probably nitpicking here, but this small discrepancy may still confuse some users.

* With the `if` block, the 4th star is now entirely filled on pan release:
![after](https://cloud.githubusercontent.com/assets/241553/26779861/2c40c58e-49f8-11e7-8a1c-cc759d3aefba.gif)

And here is `<Rating showRating fractions={1} startingValue={3.3} />`:
![fractions](https://cloud.githubusercontent.com/assets/241553/26780040/e8cd1a2c-49f8-11e7-8859-6dd9b4e0a779.gif)